template/jest.config.js

```diff
@@ -1,13 +1,22 @@
 module.exports = {
-	// include testing-library matchers
-	setupFilesAfterEnv: ['@testing-library/jest-dom/extend-expect'],
+  preset: "ts-jest",
```

Comment:
while the project doesn't include typescript right right now, this was easy to add, and will help when we do want to additionally support typescript as a possible generated output.